app/views/govuk_publishing_components/component_guide/component_doc/_component_output.html.erb

```diff
@@ -1,15 +1,15 @@
 <% component = capture do %>
   <% if example.has_block? %>
     <%= render component_doc.partial_path, example.html_safe_data do %>
-      <%= ERB.new(example.block).result(binding).html_safe %>
+      <%= render inline: example.block %>
```

Comment:
Is the previous commit needed if we're changing this line again in this commit?

Reply:
Thanks for looking Emma.

Yeah I think this is useful as two commits. First one makes the change that we render the block. Second one changes the approach in multiple places.